assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -39,60 +90,118 @@ export default class AiLlmsListEditor extends Component {
-                <tr data-persona-id={{llm.id}} class="ai-llm-list__row">
-                  <td><strong>{{llm.display_name}}</strong></td>
-                  <td>{{i18n
-                      (concat "discourse_ai.llms.providers." llm.provider)
-                    }}</td>
-                  <td>
-                    <DToggleSwitch
-                      @state={{llm.enabled_chat_bot}}
-                      {{on "click" (fn this.toggleEnabledChatBot llm)}}
-                    />
-                  </td>
-                  <td>
-                    <LinkTo
-                      @route="adminPlugins.show.discourse-ai-llms.show"
-                      current-when="true"
-                      class="btn btn-text btn-small"
-                      @model={{llm}}
-                    >{{i18n "discourse_ai.llms.edit"}}</LinkTo>
-                  </td>
+        {{#if this.hasLlmElements}}
+          <section class="ai-llms-list-editor__configured">
+            <div class="admin-page-subheader">
+              <div class="admin-page-subheader__title-row">
+                <h2 class="admin-page-subheader__title">
+                  {{i18n "discourse_ai.llms.configured.title"}}
+                </h2>
+              </div>
+            </div>
+            <table>
+              <thead>
+                <tr>
+                  <th>{{i18n "discourse_ai.llms.display_name"}}</th>
+                  <th>{{i18n "discourse_ai.llms.provider"}}</th>
+                  <th>{{i18n "discourse_ai.llms.enabled_chat_bot"}}</th>
+                  <th></th>
                 </tr>
-              {{/each}}
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                {{#each @llms as |llm|}}
+                  <tr data-persona-id={{llm.id}} class="ai-llm-list__row">
+                    <td class="column-name">
+                      <h3>{{llm.display_name}}</h3>
+                      <p>
+                        {{i18n
+                          (concat
+                            "discourse_ai.llms.model_description."
+                            llm.provider
+                            "-"
+                            llm.name
+                          )
+                        }}
+                      </p>
+                    </td>
+                    <td>
+                      {{i18n
+                        (concat "discourse_ai.llms.providers." llm.provider)
+                      }}
+                    </td>
+                    <td>
+                      <DToggleSwitch
+                        @state={{llm.enabled_chat_bot}}
+                        {{on "click" (fn this.toggleEnabledChatBot llm)}}
+                      />
+                    </td>
+                    <td class="column-edit">
+                      <LinkTo
+                        @route="adminPlugins.show.discourse-ai-llms.show"
+                        class="btn btn-default"
+                        @model={{or llm.id llm.llm.id}}
```

Comment:
I can't check code locally for now, but I looked at the code quickly.

Can you show me what `console.log(this.args.model)` gives at this line https://github.com/discourse/discourse-ai/blob/main/assets/javascripts/discourse/components/ai-llm-editor-form.gjs#L95 please?

Reply:
Maybe I borked the routing a bit when I separated that LLM route? Here is what the console.log gives:

![image](https://github.com/user-attachments/assets/c237bb0d-042c-46bf-924f-4d459dd1ee58)

Reply:
I fixed this in my commit ... honestly I think it may be an issue with llms hallucinating code cause it had ai_persona there ... which is totally unrelated.